### PR TITLE
Added constant values for SSZUint64 and UInt64

### PR DESCRIPTION
**Pull request by rolfyone**
We've got a heavy amount of duplication with 32GWEI, and zero was already fairly well handled in UInt64, but added similar logic in SSZUInt64.

## Documentation

- [X] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [X] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/primitive/SszUInt64.java [hunk 1]
```diff
@@ -19,9 +19,23 @@
...
-  public static final SszUInt64 ZERO = SszUInt64.of(UInt64.ZERO);
+  public static final SszUInt64 ZERO = SszUInt64.valueOf(UInt64.ZERO);
+  public static final SszUInt64 THIRTY_TWO_GWEI = SszUInt64.valueOf(UInt64.THIRTY_TWO_GWEI);
+
+  public static final SszUInt64 MAX_VALUE = SszUInt64.valueOf(UInt64.MAX_VALUE);
 
   public static SszUInt64 of(UInt64 val) {
+    if (val.isZero()) {
+      return ZERO;
+    } else if (val.isThirtyTwoGwei()) {
```
Comment (tbenr) [inline infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/primitive/SszUInt64.java]: you don't need `else` here

Comment (rolfyone) [reply infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/primitive/SszUInt64.java]: true enough i could just do if, return etc

File: infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/primitive/SszUInt64.java [hunk 2]
```diff
@@ -19,9 +19,25 @@
...
+  public static final SszUInt64 ZERO = SszUInt64.valueOf(UInt64.ZERO);
+  public static final SszUInt64 THIRTY_TWO_ETH = SszUInt64.valueOf(UInt64.THIRTY_TWO_ETH);
+
+  public static final SszUInt64 MAX_VALUE = SszUInt64.valueOf(UInt64.MAX_VALUE);
 
   public static SszUInt64 of(UInt64 val) {
+    if (val.isZero()) {
+      return ZERO;
+    }
+    if (val.isThirtyTwoGwei()) {
```
Comment (siladu) [inline infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/primitive/SszUInt64.java]:
```suggestion
    if (val.isThirtyTwoEth()) {
```

;)

**Review by tbenr — APPROVED** [review]
LGTM
LFG

**Pull request merged**